client-src/Main.elm

```diff
@@ -442,6 +446,12 @@ update_Http_GetTermTypesAndTypeDecls2 ( termTypes, types ) model =
     )
 
 
+jumpToTop : Cmd Message
+jumpToTop =
+    Dom.setViewport 0.0 0.0
+        |> Task.perform (\_ -> NoOp)
```

Comment:
I wish I would have a better solution than a noop, but I'm not aware of how else to perform a task and then just don't do anything.